tests/hpc/install.pm

```diff
@@ -30,9 +30,8 @@ sub run() {
     script_run 'systemctl stop packagekit.service';
     # hpc channels
     my $arch     = get_var('ARCH');
-    my $build    = get_var('BUILD_HPC');
     my $reponame = "SLE-Module-HPC12";
-    assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/GA/images/repo/SLE-12-Module-HPC-POOL-$arch-Build$build-Media1/ $reponame";
+    assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/GA/images/repo/SLE-12-Module-HPC-POOL-$arch-Media1/ $reponame";
```

Comment:
Please ensure the test runs in a way that re-running the test will test the same build as expected